hibernate-reactive-core/src/test/java/org/hibernate/reactive/CompositeIdManyToOneTest.java

```diff
@@ -71,7 +80,7 @@ public static class ShoppingItem implements Serializable {
 //        @Id Long groceryListId;
         @Id private String itemName;
 
-        @Id @ManyToOne
+        @Id @ManyToOne(fetch = FetchType.LAZY)
```

Comment:
I get that this fixes the test, but I don't think we should change the mapping for it.

We should at least replace the test with an equivalent one that uses the default or create an issue to make sure that we test the original use case.

I think that what I'm doing now will fix this test. Maybe we can wait merging this fix for now

Reply:
Sure, no problem